docs/getting-started/brokers/django.rst

```diff
@@ -37,6 +37,10 @@ configuration values.
 .. code-block:: console
 
     $ python manage.py syncdb
+    
+ Or if you are using Django 1.9 or later
```

Comment:
I would put this first, and leave the `syncdb` for Django < 1.7 (is this even supported any more?). `migrate` was added in 1.7.